pages/admin/acessos-diarios.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+  <div class="daily-access">
+    <div class="daily-access__time-container">
+      <v-dialog
+        ref="dialogStartDate"
+        v-model="startDateDialog"
+        :return-value.sync="startDate"
+        persistent
+        width="290px"
+      >
+        <template #activator="{ on, attrs }">
+          <v-text-field
+            v-model="startDate"
+            class="daily-acces__input"
+            clearable
+            placeholder="Período de início"
+            label="Período de início"
+            v-bind="attrs"
+            solo
+            v-on="on"
+          />
+        </template>
+        <v-date-picker v-model="startDate" scrollable>
+          <v-spacer />
+          <v-btn text color="primary" @click="startDateDialog = false">
+            Cancelar
+          </v-btn>
+          <v-btn text color="primary" @click="$refs.dialogStartDate.save(startDate)">
+            OK
+          </v-btn>
+        </v-date-picker>
+      </v-dialog>
+      <v-dialog
+        ref="dialogEndDate"
+        v-model="endDateDialog"
+        :return-value.sync="endDate"
+        persistent
+        width="290px"
+      >
+        <template #activator="{ on, attrs }">
+          <v-text-field
+            v-model="endDate"
+            class="daily-acces__input"
+            clearable
+            placeholder="Período final"
+            label="Período final"
+            readonly
+            v-bind="attrs"
+            solo
+            v-on="on"
+          />
+        </template>
+        <v-date-picker v-model="endDate" scrollable>
+          <v-spacer />
+          <v-btn text color="primary" @click="endDateDialog = false">
+            Cancelar
+          </v-btn>
+          <v-btn text color="primary" @click="$refs.dialogEndDate.save(endDate)">
+            OK
+          </v-btn>
+        </v-date-picker>
+      </v-dialog>
+      <v-btn color="primary" large @click="executeSearch()">
+        BUSCAR
+      </v-btn>
+    </div>
+    <div
+      v-if="series && series.length > 0"
+      class="daily-access__graph-wrapper"
+    >
+      <VueApexCharts
+        :options="chartOptions"
+        :series="series"
+      />
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  name: 'AdminDailyAccess',
+  components: {
+    VueApexCharts: () => import('vue-apexcharts')
+  },
+  layout: 'admin',
+  data () {
+    return {
+      startDate: null,
+      startDateDialog: false,
+      endDate: new Date(Date.now() - new Date().getTimezoneOffset() * 60000)
+        .toISOString()
+        .substr(0, 10),
+      endDateDialog: false,
+
+      // CHART OPTIONS
+      series: [],
+      chartOptions: {
```

Comment:
acho que mais pra frente uma melhoria seria definir esse objeto em um utils ou algo do tipo, pode facilitar na hora de exibir outros tipos de gráficos também

Reply:
Podemos deixar tbm, mas como os gráficos vão ser meio diferentes uns dos outros ainda vale a pena?

Reply:
se o objeto for muito diferente um do outro acho que não, mas se for algo que mude só o "type" etc sim